egui_glow/src/misc_util.rs

```diff
@@ -197,7 +197,7 @@ pub(crate) unsafe fn need_to_emulate_vao(gl: &glow::Context) -> bool {
         ));
         if version_str.contains("1.0") {
             //need to test OES_vertex_array_object .
-            gl.supported_extensions()
+            !gl.supported_extensions()
```

Comment:
easy mistake to make. I think it would be less confusing if the function would be called `supports_vao`. As written, it leads to double-negatives like this.

Reply:
Ok i fixed it.